tests/resource/model_resource.py

```diff
@@ -3,97 +3,101 @@
 
 .. moduleauthor:: Peter Morihladko <morihladko@atteq.com>
 """
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *
```

Comment:
The most important thing is to start unit tests on various Python environments Py2.7, Py3.5 and Py3.6. Yes, that is right. However, the old sources should be/are working as well. We'll see how we deal with the migration, when we set up the building on Travis.